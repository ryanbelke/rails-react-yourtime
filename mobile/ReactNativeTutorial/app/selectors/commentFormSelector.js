export default state => state.get('commentForm');
