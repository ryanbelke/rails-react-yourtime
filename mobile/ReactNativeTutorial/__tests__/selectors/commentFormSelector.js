import { fromJS } from 'immutable';
import commentFormSelector from 'ReactNativeTutorial/app/selectors/commentFormSelector';

describe('commentFormSelector', () => {
  it('selects the comment form store', () => {
    const state = fromJS({
      commentForm: {
        author: 'Alexey',
        text: 'Random comment',
      },
    });
    expect(commentFormSelector(state)).toMatchSnapshot();
  });
});
