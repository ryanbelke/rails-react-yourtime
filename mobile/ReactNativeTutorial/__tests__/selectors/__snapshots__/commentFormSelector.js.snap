exports[`commentFormSelector selects the comment form store 1`] = `
Object {
  "author": "Alexey",
  "text": "Random comment",
}
`;
